packages/gitbook-docs/src/__tests__/parseContent.test.ts

```diff
@@ -119,13 +115,13 @@ describe('parseContent', () => {
   it('should preserve a single space into a td tag nested into other tags', () => {
     expect(
       parseContent(
-        '<td><strong>before space</strong> <a><strong>after space</strong></a></td>',
+        '<td><strong>before space</strong> <a href="target-link"><strong>after space</strong></a></td>',
```

Comment:
Do you need this test? If you are testing anchor parsing then create a specific test